packages/api-browser/rollup.config.js

```diff
@@ -1,3 +1,24 @@
+// import resolve from 'rollup-plugin-node-resolve';
+// import commonjs from 'rollup-plugin-commonjs';
+
+// export default {
+//   entry: 'bundle.js',
+//   moduleName: 'ssf',
+//   format: 'umd',
+//   plugins: [
+//     resolve({
+//       module: true,
+//       main: true,
+//       browser: false,
+//       jsnext: true
+//     }),
+//     commonjs({
+//       include: '**'
+//     })
+//   ],
+//   dest: 'build/dist/containerjs-api.js'
+// };
+
 import resolve from 'rollup-plugin-node-resolve';
```

Comment:
Oops. Good spot! Not sure what you mean by bundle api-utility? The node resolver includes any packages installed in node_modules, so that would include api-utility. Or have I mis-understood? :-)